string nameStudent
int32 studentCode
float32 GPA
